### ISSUE TITLE
Rails 5 support

### ISSUE THREAD

**Issue by Eilie** [rikkipitt/rails_admin_history_rollback#13]
```
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 5.0.0.rc1)

  In Gemfile:
    rails (< 5.1, >= 5.0.0.rc1)

    rails_admin was resolved to 0.8.1, which depends on
      rails (< 6, >= 4.0)

    rails_admin_history_rollback (~> 0.0.7) was resolved to 0.0.7, which depends on
      rails (~> 4.2.0)
```

**Comment by rikkipitt** [comment]
Hi @Eilie, thanks for reporting. This is related to #11. Once RailsAdmin supports Rails v5, I'll push an update to fix this.

Best,
Rikki

**Comment by Eilie** [comment]
@rikkipitt Thank you!

**Comment by rikkipitt** [comment]
@Eilie I've just released version 0.0.8 which should work on Rails 5. Just waiting for rails_admin to release a new version to work with it too...
